requests/src/requests/Requester.scala

```diff
@@ -331,7 +332,11 @@ case class Requester(verb: String,
             else connection.getErrorStream
 
           def processWrappedStream[V](f: java.io.InputStream => V): V = {
-            if (stream != null) {
+            // The HEAD method is identical to GET except that the server
+            // MUST NOT return a message-body in the response.
+            // https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html section 9.4
+            if (verb.toLowerCase == "head") f(new ByteArrayInputStream(Array()))
```

Comment:
The HTTP verbs are case-sensitive. Probably you need to check for `HEAD` because `head` or `Head` are considered extension-methods and don't share the `HEAD` semantics.
```suggestion
            if (verb == "HEAD") f(new ByteArrayInputStream(Array()))
```

Reply:
@lolgab updated as per your suggestion